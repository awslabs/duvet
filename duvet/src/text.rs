// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

mod find;
pub mod view;

pub use find::find;
pub use view::view;
