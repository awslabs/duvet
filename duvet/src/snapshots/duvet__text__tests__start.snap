---
source: src/text.rs
expression: "find(\"a\", \"a b c d\")"
---
Some(
    (
        0..1,
        "a",
    ),
)
